app/views/candidate_interface/degrees/grade/_form_fields.html.erb

```diff
@@ -13,7 +13,7 @@
       <%= f.govuk_text_field(:other_grade, text_field_options) %>
     <% end %>
     <% @international_main_grades.each_with_index do |grade, i | %>
```

Comment:
I think we can lose the `each_with_index` here now?

Reply:
Good catch!